codespell_lib/data/dictionary.txt

```diff
@@ -29711,7 +29772,14 @@ instatiates->instantiates
 instatiating->instantiating
 instatiation->instantiation
 instatiations->instantiations
+insteall->install, instead,
```

Comment:
Any `insteall->instill` (and friends) misspellings?

Reply:
I couldn't find this typo.